examples/arithmetics/src/language-server/generated/ast.ts

```diff
@@ -115,7 +115,7 @@ export function isNumberLiteral(item: unknown): item is NumberLiteral {
 
 export type ArithmeticsAstType = 'AbstractDefinition' | 'BinaryExpression' | 'DeclaredParameter' | 'Definition' | 'Evaluation' | 'Expression' | 'FunctionCall' | 'Module' | 'NumberLiteral' | 'Statement';
 
-export type ArithmeticsAstReference = 'FunctionCall:func';
+export type ArithmeticsAstReference = 'PrimaryExpression:func';
```

Comment:
So the name preceding the colon is now the parser rule, not the type?

Reply:
Yes, exactly. This makes cross references unambiguous in regards to the actual type of the object we're passing around.